### PR TITLE
Oculus Quest OpenVR support

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
# XRTK - Mixed Reality Toolkit Change Request

## Overview

Adding Oculus Quest OpenVR support

## Target of the change:

Is this enhancement for:

- Platform (new or updated platform)

## Changes:

Brief list of the targeted features that are being changed.

- New controller definition
- Platform updates

### CODE REVIEW

File: XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/OpenVR/OculusQuestControllerMappingProfile.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (StephenHodgson) [inline XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/OpenVR/OculusQuestControllerMappingProfile.cs]: Are these copied from the mrtk?

Comment (StephenHodgson) [inline XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/OpenVR/OculusQuestControllerMappingProfile.cs]: Are these copied from the mrtk?

File: XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/UnityInput/GenericJoystickController.cs [hunk 2]
```diff
@@ -150,6 +150,7 @@ protected void UpdateSingleAxisData(MixedRealityInteractionMapping interactionMa
                     // If our value was updated, raise it.
                     if (interactionMapping.Updated)
                     {
+                        Debug.LogError($"Interaction updated {interactionMapping.Description} - Value [{interactionMapping.FloatData}]");
```
Comment (StephenHodgson) [inline XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/UnityInput/GenericJoystickController.cs]: Why are these errors here?

Just to verify you're getting data?

**Comment by SimonDarksideJ** [comment]
Pointer issue resolved in the latest SDK PR, turns out to be a mapping issue in the profile
https://github.com/XRTK/SDK/pull/49

**Comment by SimonDarksideJ** [comment]
Closing as #234 now implements this more effectively

**Pull request closed without merging**